docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -11,12 +11,12 @@ You can install the {ProductShortName} plugin in the Ultimate and the Community
 .Prerequisites
 include::snippet_jdk-hardware-mac-prerequisites.adoc[]
 
-* The latest version of `mta-cli` from the link:{MTADownloadPageURL}[{ProductShortName} download page]
+* The latest version of `mta-cli` from the link:{{ProductShortName}DownloadPageURL}[{ProductShortName} download page]
 
 .Procedure
 
 . In IntelliJ IDEA, click the *Plugins* tab on the Welcome screen.
 . Enter `{ProductName}` in the Search field on the *Marketplace* tab.
-. Select the *Migration Toolkit for Applications (MTA) by Red Hat* plugin and click *Install*.
+. Select the *{ProductName} ({ProductShortName}) by Red Hat* plugin and click *Install*.
```

Comment:
Did QE check that there is a plugin named "Migration Toolkit for Runtimes (MTR) by Red Hat" ?